src/SFA.DAS.EmployerIncentives.Domain/ValueObjects/IncentivesConfiguration.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using SFA.DAS.EmployerIncentives.Abstractions.Domain;
+using SFA.DAS.EmployerIncentives.Domain.ApprenticeshipIncentives.Exceptions;
+using SFA.DAS.EmployerIncentives.Enums;
+
+namespace SFA.DAS.EmployerIncentives.Domain.ValueObjects
+{
+    public class IncentivesConfiguration
```

Comment:
This is not required and is not a business object.  There is already an IncentivePaymentProfile that does this.

Configuration is an infrastructure concern.

Reply:
renamed